niworkflows/interfaces/cifti.py

```diff
@@ -339,7 +340,7 @@ def _create_cifti_image(
     bold_img = _reorient_image(bold_img, orientation="LAS")
     label_img = _reorient_image(label_img, orientation="LAS")
 
-    bold_data = bold_img.get_fdata(dtype="float32")
+    bold_data = bold_img.get_fdata()
```

Comment:
Is there a good reason to upcast to float64?

Reply:
nope, can revert that